modules/ballerina-core/src/main/java/org/ballerinalang/util/semantics/SemanticAnalyzer.java

```
@@ -2610,7 +2610,12 @@ private void visitLExprsOfAssignment(AssignStmt assignStmt, Expression[] lExprs)
 
                 SimpleVarRefExpr refExpr = (SimpleVarRefExpr) expr;
                 String varName = refExpr.getVarName();
```

We should throw an error, if both variables have been already declared.